tests/models.py

```diff
@@ -8,6 +11,15 @@ class SuperVillain(models.Model):
     stats = JSONField(default=None, blank=True, null=True)
 
 
+class Minion(models.Model):
+    _PREFIX = 'I can: '
+
+    name = models.CharField(max_length="20", default="Igor")
+    skills = JSONField(default=None, blank=True, null=True,
```

Comment:
This would be a good place to test the default order (by taking out the named arguments and testing before and after the serializer kwargs).